docs/blog/write-the-docs-2018-kickoff.rst

```diff
@@ -0,0 +1,77 @@
+
+If you want to learn more about the organization behind the community, you can read :doc:`About Us </about-us>`. You can also find a high-level overview of the teams and tasks in :doc:`Team </team>`.
+
+Job Board
+---------
+
+As we mentioned in previous newsletters and Slack posts, we are launching a shiny new job board! Led by Kelly O'Brien, the job board aims to make it easy for documentarians to find their dream gigs, and for employers to find their dream documentarians.
+
+We're currently in the beta-testing phase, so if you're interested in helping us test the board – either as an employer or an applicant – we'd love to hear from you at `jobs@writethedocs.org <mailto:jobs@writethedocs.org>`_.
+
+Note: In order to help optimize our job-related efforts, we looked at our current communication channels, and came to the conclusion that the #jobs Slack channel together with the job board is the most likely to be a successful combination. Therefore, we are sadly closing the Write the Docs forum on (date?).
+
+Swag
+----
+
+Some of you might have noticed that this year's conference tickets do not include shirts. We realized that after 5 years of printing shirts for multiple conferences, the swag offering could use a bit of a refresh. In addition, the overhead of printing hundreds of shirts and managing inventory simply proved ineffective over time.
+
+So this year we are changing swag tactics, and instead of including shirts with the ticket purchase, we happily present **tote bags**! No sizing required, smoother production workflow, and just as collectible :)
+
+But have no fear, you can still get a shirt for your favorite conference! Inspired by the likes of DjangoCon US, we will team up with `TeeSpring <http://teespring.com/>`_, a trusted print-on-demand provider that works on a campaign-based model.
+
+Over the next few weeks we will be setting up a campaign where you can order a shirt with this year's conference design, and anyone who orders by a certain date should receive their shirt in time for the conference.
+
+We will continue to print speaker and staff hoodies, since we can estimate sizing and quanitities much more accurately than we can with shirts.
+
+Community-Run Events
+--------------------
+
+In addition to the official conferences that are organized by our team and carry the formal Write the Docs brand, we have seen a growing number of events that are organized by our community members.
+
+We are thrilled that our community is excited to share knowledge and network with like-minded folks, and happy to support and cross-promote these events in our community.
+
+To clarify the difference between a core Write the Docs event and a community-run event, and to answer questions for those who wish to start a community event, we created :doc:`Community-Run Events </organizer-guide/community-events>`.
```

Comment:
I think we should mention this is still something we aren't totally sure how to handle, and will be experimenting with it. Pointing to that doc as the "final decision" feels a bit off to me, given all our uncertainty.